api/src/index.ts

```diff
@@ -76,6 +86,7 @@ function createApp(postgraphileOptions: PostGraphileOptions) {
     })
   );
   app.use(postgraphile(getDbUrl("user"), "ctfnote", postgraphileOptions));
+  app.use("/calendar.ics", icalRoute(pool));
```

Comment:
I see the pool is *only* used for the ical route.
Can we use this for postgraphile too ? It would be nice to share the same connection.